CHANGES.rst

```diff
@@ -4,6 +4,21 @@ Changelog for zest.releaser
 6.8 (unreleased)
 ----------------
 
+- Allow ``pypirc`` with just a ``pypi`` section.  Previously, we
+  required either a ``server-login`` section with a ``username``
```

Comment:
`[server-login]` with square brackets to make it clearer that it is a section? (Same in next line).